host/xtest/xtest_benchmark_2000.c

```diff
@@ -78,3 +79,61 @@ ADBG_CASE_DEFINE(XTEST_TEE_BENCHMARK_2002, xtest_tee_benchmark_2002,
 		/* How to implement */ ""
 		);
 
+
+/* ----------------------------------------------------------------------- */
+/* -------------------------- AES Benchmarks ----------------------------- */
+/* ----------------------------------------------------------------------- */
+
+static void xtest_tee_benchmark_2011(ADBG_Case_t *c)
+{
+	UNUSED(c);
+
+	int mode = TA_AES_ECB;	/* AES mode */
+	int decrypt = 0; /* Encrypt */
+	int keysize = AES_128;
+	size_t size = 1024;	/* Buffer size */
```

Comment:
I know, but what about readability?
It's more convenient (for both reading and changing code), when you see each value per line (with readable and clear variable names). If we put values 1024, 0 etc. directly into the function call, I think people will every time refer to prototype of function to understand what 1024 and 0 means